helmfile/50-applications.yaml

```
@@ -109,7 +109,21 @@ releases:
   installed: {{ .Values.calicoFelixMetrics.enabled }}
   missingFileHandler: Error
 
-
+# Blackbox-sc-wc
+- name: blackbox
+  namespace: monitoring
+  labels:
+    app: blackbox
+  chart: ./charts/blackbox
+  version: 0.1.0
+  missingFileHandler: Error
+  values:
+{{ if eq .Environment.Name "service_cluster" }}
+  - values/blackbox.yaml.gotmpl
```

Could you rename this file as well?
```suggestion
  - values/blackbox-sc.yaml.gotmpl
```